packages/google_sign_in/lib/src/device_flow_widget.dart

```diff
@@ -33,43 +33,102 @@ void showDeviceFlowWidget({
     context: navigatorKey.currentContext!,
     barrierDismissible: false,
     builder: (BuildContext context) {
+      final TextStyle bodyStyle =
+          Theme.of(context).textTheme.bodyLarge!.copyWith(color: Colors.black);
+      final TextStyle titleStyle = Theme.of(context)
+          .textTheme
+          .titleLarge!
+          .copyWith(color: Colors.black, fontWeight: FontWeight.bold);
+
       return AlertDialog(
-        title: const Text('Google Sign In'),
-        content: Column(
-          mainAxisAlignment: MainAxisAlignment.center,
-          children: <Widget>[
-            const Text('From a PC, phone, or tablet go to:'),
-            Text(
-              '$verificationUrl',
-              textScaleFactor: 1.5,
-            ),
-            const Text('Then enter the code below:'),
-            Text(
-              code,
-              textScaleFactor: 2.5,
-            ),
-            const Text('Code will expire in:'),
-            _CountdownTimer(
-              expiresIn,
-              onFinished: () {
-                onExpired?.call();
-                closeDeviceFlowWidget();
-              },
-            ),
-          ],
+        scrollable: true,
+        contentPadding: const EdgeInsets.all(10),
+        content: Container(
+          padding: const EdgeInsets.symmetric(horizontal: 15, vertical: 25),
+          decoration: BoxDecoration(
+              border: Border.all(color: Colors.grey),
+              borderRadius: const BorderRadius.all(Radius.circular(5))),
+          child: Column(
+            mainAxisSize: MainAxisSize.min,
+            children: <Widget>[
+              Image.asset(
+                'packages/google_sign_in_tizen/assets/images/google.png',
+              ),
```

Comment:
or maybe
```suggestion
              const Image(
                image: AssetImage(
                  'assets/images/google.png',
                  package: 'google_sign_in_tizen',
                ),
              ),
```

I don't know which is a preferred way.

By the way, can we use the logo without Google's approval?

Reply:
> I don't know which is a preferred way.

I don't think they have a preferred way when using assets in the package itself:
- https://docs.flutter.dev/development/ui/assets-and-images#from-packages
- https://docs.flutter.dev/development/ui/assets-and-images#bundling-of-package-assets

But the first one (the one you suggested) is more clear on where the asset comes from, I'll fix it.

> By the way, can we use the logo without Google's approval?

I've actually read this document incorrectly and we can't use the "Google" logo without their permission, we can only use the "G" logo with the text "Sign In With" for buttons. I'll check the Google's [Brand Resource Center](https://about.google/brand-resource-center/brand-elements/).